### PR TITLE
Update wiki page of GBC 2023 Autumn

### PR DESCRIPTION
**Pull request by yobrevelc**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] mappools
- [x] badge prizes

### CODE REVIEW

**Review by RushFTK — APPROVED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 1]
```diff
@@ -159,6 +163,183 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+### QQ Division Grand Finals
+
+Detailed statistics for this round can be found [here](https://docs.qq.com/sheet/DTU92VHRFUm1nTm9Z?tab=xiki0i).
+
+Saturday, 16 December 2023:
+
+| Team A |  |  | Team B | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Q3: 兰州拉面专业培训班** | **7** | 3 | Q16: DWG.BurgerkingReisen | [#1](https://osu.ppy.sh/community/matches/111749143) |
+| **Q12: Team 12** | **7** | 5 | Q3: 兰州拉面专业培训班 | [#2](https://osu.ppy.sh/community/matches/111750301) |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:197]: the match links should all be `#1`. it's just a formatting thing. if there are two multiplayer lobbies for one match, it'd be `[#1](), [#2]()`

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 2]
```diff
@@ -159,6 +163,183 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+
+### Discord Division Finals
+
+Detailed statistics for this round can be found [here](https://docs.qq.com/sheet/DTU92VHRFUm1nTm9Z?tab=wmnmix).
+
+Saturday, 9 December 2023:
+
+| Team A |  |  | Team B | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **D2: viets sing** | **0** | -1 | D1: ChatGBC | Win by default |
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:286]: on all of these

```suggestion
| **D2: viets sing** | **0** | -1 | D1: ChatGBC | *win by default* |
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 3]
```diff
@@ -244,8 +425,8 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
 ### Stage instructions
 
 1. Following the Qualifier stage, all players will participate in a phase to form teams, called the *drafting phase*. Both divisions will have independent drafting phases. All unpicked players will be eliminated from the tournament.
-2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in total, and the others get eliminated from the tournament.
-3. Following the regular stage, the elimination stage will be played in both divisions, called *playoffs*. The top 8 teams from the regular stage will advance to a winner bracket, other teams will get into loser bracket.
+2. Following the drafting phase, a Swiss stage will be played in both zones, called the *regular stage*. 16 teams will advance in both zones, and the others get eliminated from the tournament.
+3. Following the regular stage, the elimination stage will be played in both divisions, called *playoffs*. The top 8 teams from the regular stage in each zones will advance to a winner bracket, other teams will get into loser bracket.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:429]:
```suggestion
3. Following the regular stage, the elimination stage will be played in both divisions, called *playoffs*. The top 8 teams from the regular stage in each zone will advance to a winner bracket, other teams will get into loser bracket.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 4]
```diff
@@ -332,20 +513,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
 ### Mappool instructions
 
-1. There will be five separate mappools. *This might change depending on the number of incoming registrations.*
+1. There will be nine separate mappools.
    - Qualifiers will use a separate mappool.
-   - Regular stage round 1 and round 2 use mappool #1.
-   - Regular stage round 3 and round 4 use mappool #2.
-   - Quarterfinals and Semifinals use mappool #3.
-   - Finals and Grand Finals use mappool #4.
+   - From the mappool #1 in regular stage round 1, every mappool will change half of the pool from the previous one. Mappool #2 will change all the RC map from mappool #1, and mappool #3 will change all the HB + LN map from mappool #2. This process will repeat until the last mappool.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:518]:
```suggestion
   - From the mappool #1 in regular stage round 1, every mappool will change half of the pool from the previous one. Mappool #2 will change all the RC maps from mappool #1, and mappool #3 will change all the HB + LN map from mappool #2. This process will repeat until the last mappool.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 5]
```diff
@@ -332,20 +513,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+   - SV map will also follow that rules, mappool #2 will change SV1, mappool #3 will change SV2, so on until the last mappool.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:519]:
```suggestion
   - SV maps also follow the same rule: mappool #2 will change SV1, mappool #3 will change SV2, and so on until the last mappool.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 6]
```diff
@@ -332,20 +513,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+   - TB map will always be changed in each mappools.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:520]:
```suggestion
   - TB maps will always be changed in each mappool.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 7]
```diff
@@ -332,20 +513,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+   - Per Quarterfinals mappool size expansion, there will be an extra RC map released in that mappool than round 4.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:521]:
```suggestion
   - Due to the Quarterfinals mappool size expansion, there will be an extra RC map in that mappool compared to round 4.
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 8]
```diff
@@ -332,20 +513,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
 2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Gameplay/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
 3. The mappool sizes are as follows:
-   - Qualifiers: 8 beatmaps
-   - Mappool #1 & #2: 15 beatmaps
-   - Mappool #3 & #4: 17 beatmaps.
-4. Each mappool has one tiebreaker, except for the Qualifiers.
+   - Qualifiers: 8 beatmaps
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:524]:
```suggestion
   - Qualifiers: 8 beatmaps
```

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 9]
```diff
@@ -332,20 +513,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+   - Per Quarterfinals mappool size expansion, there will be an extra RC map released in that mappool than round 4.
 2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Gameplay/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
 3. The mappool sizes are as follows:
-   - Qualifiers: 8 beatmaps
-   - Mappool #1 & #2: 15 beatmaps
-   - Mappool #3 & #4: 17 beatmaps.
-4. Each mappool has one tiebreaker, except for the Qualifiers.
+   - Qualifiers: 8 beatmaps   
+   - Mappool in regular phase: 15 beatmaps
+   - Mappool in elimination phase: 17 beatmaps.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:526]:
```suggestion
   - Mappool in the regular phase: 15 beatmaps
   - Mappool in the elimination phase: 17 beatmaps.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/GBC/GBC_2023_Autumn/en.md [hunk 10]
```diff
@@ -332,20 +517,19 @@ The GB Cup 2023 Autumn is run by Team GB and various community members.
...
+1. There will be nine separate mappools.
    - Qualifiers will use a separate mappool.
-   - Regular stage round 1 and round 2 use mappool #1.
-   - Regular stage round 3 and round 4 use mappool #2.
-   - Quarterfinals and Semifinals use mappool #3.
-   - Finals and Grand Finals use mappool #4.
+   - From the mappool #1 in regular stage round 1, every mappool will change half of the pool from the previous one. Mappool #2 will change all the RC maps from mappool #1, and mappool #3 will change all the HB + LN map from mappool #2. This process will repeat until the last mappool.
+   - SV maps also follow the same rule: mappool #2 will change SV1, mappool #3 will change SV2, and so on until the last mappool.
+   - TB maps will always be changed in each mappool.
+   - Due to the Quarterfinals mappool size expansion, there will be an extra RC map in that mappool compared to round 4.
```
Comment (Walavouchey) [inline wiki/Tournaments/GBC/GBC_2023_Autumn/en.md:525]: following explanation on discord

```suggestion
   - The mappool size expands by 1 slot in the Quarterfinals, adding one extra RC map to the pool. For this round, the rest of the RC maps are kept from the previous round, while the HB + LN maps are switched to new ones.
```

**Review by RushFTK — APPROVED** [review]

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**